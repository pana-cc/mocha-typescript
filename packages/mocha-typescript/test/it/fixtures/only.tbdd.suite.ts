import { suite, test } from "../../../index";

suite("vanila tdd suite", () => {
    test("vanila test", () => {});
    test.only("only", () => {});
});

describe("vanila bdd suite", () => {
    it("vanila test", () => {});
    it.only("only", () => {});
});
