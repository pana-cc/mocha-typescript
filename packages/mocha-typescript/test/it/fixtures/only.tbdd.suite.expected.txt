
  vanila tdd suite
    âœ“ only

  vanila bdd suite
    âœ“ only

  2 passing
